builtin/logical/pki/backend.go

```diff
@@ -192,6 +200,12 @@ type backend struct {
 	tidyStatusLock sync.RWMutex
 	tidyStatus     *tidyStatus
 
+	certCount                           *uint32
+	revokedCertCount                    *uint32
+	certsCounted                        bool
```

Comment:
This would be better if it were an atomic boolean with how we are using it across threads at the moment.

Reply:
Yeah, that's fair, I've updated to include this.